### ISSUE TITLE
Temperature exploding while training with ASE module

### ISSUE THREAD

**Issue by aaronchen0316** [mir-group/flare#253]
Hi,

Recently, most of my training encountered issues where the MD temperature just kept increasing rapidly (e.x. the AgI system you published online, I didn't change training parameters). I am using the latest commit e2a1427. I am wondering whether this is the same issue mentioned in #232 ? The wrapped positions are copied so the predicted force is wrong.
Few things I noticed in the output file: the GP forces are huge (corresponding MAE is high), although likelihood is high, the likelihood gradient is also massive.
Thank you.

**Comment by jonpvandermause** [comment]
Hi aaronchen0316,

Could you please post your definitions of the `md_engine` and `md_dict` parameters? For example, if you're doing an ASE OTF simulation in the NVTBerendsen ensemble, you should have something like this in your input script:

```
from ase import units
temperature = 5000  # K
taut = 25 * units.fs
md_engine = 'NVTBerendsen'
md_dict = {'temperature': temperature,'taut': taut}
```

Thank you,
Jon

**Comment by YuuuuXie** [comment]
So it seems that the issue recently appears because we pushed a new commit to solve #247 . This should be addressed in the current pull request #254
